server/api.go

```diff
@@ -112,6 +190,37 @@ func (p *Plugin) handleExternalAPIjs(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (p *Plugin) proxyExternalAPIjsJaaS(w http.ResponseWriter, r *http.Request) {
```

Comment:
```suggestion
func (p *Plugin) proxyExternalAPIjsJaaS(w http.ResponseWriter, _ *http.Request) {
```
another lint failure